src/edu/csus/ecs/pc2/core/execute/SandboxResults.java

```diff
@@ -62,5 +74,9 @@ public int getExitCode() {
             }
         }
     }
+    
```

Comment:
Again, problems with GitHub's commenting rules.  This comment is about line 65 (above):  `Object obj = resultsProperties.get("exitcode").`
With all the attention given elsewhere to using constants to define hash keys, it seems at best inconsistent that a hard-coded string ("exitcode") is used here.

Reply:
Agreed. Added and now using
public static final String EXITCODE_KEY = "exitcode";